SerialPrograms/Source/PokemonRSE/PokemonRSE_Settings.cpp

```diff
@@ -0,0 +1,99 @@
+/*  Pokemon RSE Settings
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+#include "Common/NintendoSwitch/NintendoSwitch_ControllerDefs.h"
+#include "CommonFramework/Globals.h"
+
+#include "PokemonRSE_Settings.h"
+
+namespace PokemonAutomation{
+namespace NintendoSwitch{
+namespace PokemonRSE{
+
+
+
+GameSettings& GameSettings::instance(){
+    static GameSettings settings;
+    return settings;
+}
+GameSettings::GameSettings()
+    : BatchOption(LockMode::LOCK_WHILE_RUNNING)
+    , m_menu_navigation("<font size=4><b>Menu Navigation Timings:</b></font>")
+    , GAME_TO_HOME_DELAY(
```

Comment:
How many of these are relevant to GBA? Stuff like the Home menu didn't exist on the real thing.